components/clp-py-utils/clp_py_utils/initialize-orchestration-db.py

```diff
@@ -64,7 +65,8 @@ def main(argv):
                     `clp_binary_version` INT NULL DEFAULT NULL,
                     `clp_config` VARBINARY(60000) NOT NULL,
                     PRIMARY KEY (`id`) USING BTREE,
-                    INDEX `JOB_STATUS` (`status`) USING BTREE
+                    INDEX `JOB_STATUS` (`status`) USING BTREE,
+                    INDEX `LAST_UPDATE_TIME` (`update_time`) USING BTREE
```

Comment:
for consistency, we shall use the `JOB_` prefix.
```suggestion
                    INDEX `JOB_UPDATE_TIME` (`update_time`) USING BTREE
```

on a separate note, i agree "last update time" is more intuitive than "update time". if we do want to mention "last update time", it's better if we update all references. i think we can keep "update_time" as-is, since the name "update time" shouldn't be too confusing in this job table.